pkgs/applications/window-managers/windowmaker/dockapps/default.nix

```diff
@@ -0,0 +1,21 @@
+{ config, lib, pkgs }:
+
+lib.makeScope pkgs.newScope (self: with self; {
+
+  dockapps-sources = pkgs.fetchgit {
+    url = "https://repo.or.cz/dockapps.git";
+    rev = "b2b9d872ee61c9b329e4597c301e4417cbd9c3ea";
```

Comment:
Is there a branch or tag we could track?

Reply:
Not so easily. It is essentially a monorepo of a plethora of dockapps.